ui/apps/platform/cypress/helpers/formHelpers.js

```diff
@@ -40,6 +40,16 @@ export function getHelperElementByLabel(label) {
         });
 }
 
+/**
+ * Gets the parent `div` of a dt/dd description list group given the text value of the `dt`
+ * element and the text value of the `dd` element.
+ * @param {string} term The text content of the `dt` element
+ * @param {string} description The text content of the `dd` element
+ */
+export function getDescriptionListGroup(term, description) {
+    return cy.get(`div:has(dt:has(*:contains("${term}")) + dd:has(*:contains("${description}")))`);
```

Comment:
`:has` pseudo-selector in test selectors deserves a short presentation at team meeting

Reply:
👍  I'll make sure to note that this is the jQuery `:has` pseudo-selector, which functions similarly but not quite the same as the new CSS `:has`. The CSS version supports `:has` with combinator arguements and from what I can tell the jQuery version does not.

e.g. Select a `div` that has a `button` as a direct descendant: `div:has(> button)`. There doesn't seem to be a parallel in the jQuery version that Cypress uses.